TileStache/Goodies/VecTiles/transform.py

```diff
@@ -685,6 +685,32 @@ def overlap(feature_layers, base_layer, cutting_layer,
         target_attribute, cutting_attrs, keep_geom_type)
 
 
+# map from old or deprecated kind value to the value that we want
+# it to be.
+_deprecated_landuse_kinds = {
+    'station': 'substation',
```

Comment:
Train stations are in the POIs layer. And power stations are `power=plant` or `power=generator`. This filter is not applied to the POIs layer, so anything drawing from that layer will be unaffected.